src/cmds.c

```diff
@@ -715,10 +714,12 @@ static void cmd_console(struct userrec *u, int idx, char *par)
           dcc[dest].u.chat->con_flags &= ~logmodes(s);
       }
     }
+  } else if (reset) {
+    dcc[dest].u.chat->con_flags = (u->flags & USER_MASTER) ? conmask : 0;
```

Comment:
Why reset to 0 (no flags) when not a master?

Reply:
That seems to be the overall intention of the code I gathered, even ignoring the bit that was already there in .console:
https://github.com/eggheads/eggdrop/blob/0d6d8d/src/cmds.c#L1388-L1389
https://github.com/eggheads/eggdrop/blob/0d6d8d/src/cmds.c#L2238-L2239
https://github.com/eggheads/eggdrop/blob/0d6d8d/src/dcc.c#L685
https://github.com/eggheads/eggdrop/blob/0d6d8d/src/misc.c#L918-L936

Reply:
I guess unifying that into using check_conflags() is in order, but the result stays pretty much the same I think

Reply:
As a new non-master user joining partyline has no flags as well, this seems to be OK after all.